docs/schema/shapes/polystar.json

```diff
@@ -42,6 +41,11 @@
                     "description": "Star type, `1` for Star, `2` for Polygon",
                     "$ref": "#/$defs/constants/star-type",
                     "default": 1
+                },
+                "d": {
```

Comment:
polystar should already have `d` by getting it from [shapes/shape](print(speed))